packages/sdk/src/oracle/services/statemachines/setActiveRequestByTransaction.ts

```diff
@@ -18,11 +17,12 @@ export function Handlers(store: Store): GenericHandlers<Params, Memory> {
       const provider = store.read().provider(chainId);
       const receipt = await provider.getTransactionReceipt(transactionHash);
       assert(receipt, "Unable to find transaction receipt from hash: " + transactionHash);
+      const oracle = store.read().oracleService(chainId);
       const oracleAddress = store.read().oracleAddress(chainId);
       // filter out logs that originate from oracle contract
       const oracleLogs = receipt.logs.filter((log) => log.address.toLowerCase() === oracleAddress.toLowerCase());
       // decode logs using abi
-      const decodedLogs = oracleLogs.map((log) => optimisticOracle.contractInterface.parseLog(log));
+      const decodedLogs = oracleLogs.map((log) => oracle.parseLog(log));
```

Comment:
the bug was here, using the v1 interface to parse logs, now we do it based on the oracle service which is specific to the contract type